builder/ebs/fast_launch_setup.go

```diff
@@ -17,6 +19,16 @@ import (
 // in the template, as each fast-launch enablement step occurs after the
 // copy, and each region may pick their own launch template.
 type FastLaunchTemplateConfig struct {
+	// EnableFalseLaunch allows you to specify if fast launch should be
+	// enabled on a region-by-region basis
+	//
+	// If unset, this will assume the default behaviour, i.e. the other
+	// options will imply that this is true.
+	//
+	// If this is explicitly set to `false` however, fast-launch will be
+	// disabled for the specified region, and options other than the region
+	// will be ignored.
```

Comment:
Small rewrite to fix typo and help clarify the intent. 
Is the default behavior always true or based on some condition? 

```suggestion
	// Enable fast launch allows you to disable fast launch settings on the region level.
	//
	// If unset, the default region behavior will be assumed.
	// Using other fast launch options, while unset, will imply enable_fast_launch to be true.
	//
	// If this is explicitly set to `false` fast-launch will be
	// disabled for the specified region and all other options besides region
	// will be ignored.
```

Reply:
Yeah so the default behaviour is implicitly true if there's a `region_launch_templates` defined with at least one of the mandatory attributes besides `region` (refer to the [`Prepare`](https://github.com/hashicorp/packer-plugin-amazon/blob/main/builder/ebs/fast_launch_setup.go#L43) implementation for the structure for more info).
If explicitely set to false, this just disables fast-launch for the region, as is requested in the linked issue.